src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const listOfEmployees = [...document.querySelectorAll(`ul li`)];
```

Comment:
Ensure that the `ul li` selector correctly matches the intended list items. If the `li` elements are not direct children of a `ul`, this query will not work as expected.